### PR TITLE
add online icon to multiple pages replacing text "Online Content"

### PR DESCRIPTION
**Pull request by gordonleacock**
Closes #797 

Add green online content icon to partials:

- app/views/catalog/_arclight_index_default.html.erb
- app/views/catalog/_arclight_index_default.html.erb

Modified:

- spec/features/search_results_spec.rb to check for icon class
- app/assets/stylesheets/arclight/modules/search_results.scss to make icon green

### CODE REVIEW

File: app/assets/images/blacklight/compact.svg [hunk 1]
```diff
@@ -1 +1,5 @@
+<!--
```
Comment (mejackreed) [inline app/assets/images/blacklight/compact.svg]: My understanding was that this came from Material Icons. I'm not sure we need additional coments

File: app/assets/images/blacklight/list.svg [hunk 2]
```diff
@@ -1 +1,5 @@
+<!--
+  Used unmodified from https://fontawesome.com
+  CC BY 4.0 https://creativecommons.org/licenses/by/4.0/
```
Comment (mejackreed) [inline app/assets/images/blacklight/list.svg]: My understanding was that this came from @jvine not FontAwesome. I'm not sure we need additional comments

Comment (gordonleacock) [reply app/assets/images/blacklight/list.svg]: My bad. Will fix.

Comment (gordonleacock) [reply app/assets/images/blacklight/list.svg]: Done.

File: app/assets/images/blacklight/compact.svg (hunk 1 above)
Comment (gordonleacock) [reply app/assets/images/blacklight/compact.svg]: Fixed.

File: app/assets/images/blacklight/list.svg (hunk 2 above)
Comment (mejackreed) [reply app/assets/images/blacklight/list.svg]: 👍

**Comment by mejackreed** [comment]
Is this addressing #782 ?

File: app/assets/stylesheets/arclight/modules/search_results.scss [hunk 3]
```diff
@@ -57,6 +57,9 @@ article.document {
   .blacklight-icons svg {
     fill: $secondary;
   }
+  .blacklight-icons.al-online-content-icon svg {
+   fill: $green;
+ }
```
Comment (mejackreed) [inline app/assets/stylesheets/arclight/modules/search_results.scss]:
```suggestion
  }
```

File: app/assets/stylesheets/arclight/modules/search_results.scss [hunk 4]
```diff
@@ -57,6 +57,9 @@ article.document {
   .blacklight-icons svg {
     fill: $secondary;
   }
+  .blacklight-icons.al-online-content-icon svg {
+   fill: $green;
```
Comment (mejackreed) [inline app/assets/stylesheets/arclight/modules/search_results.scss]:
```suggestion
    fill: $green;
```

File: config/locales/arclight.en.yml [hunk 5]
```diff
@@ -36,7 +36,6 @@ en:
         top_group_results:
           one: Top %{count} result
           other: Top %{count} results
-      online_content_indicator: online content
```
Comment (mejackreed) [inline config/locales/arclight.en.yml]: Nice job removing this key hanging out!

**Comment by ggeisler** [comment]
@gordonleacock @mejackreed This looks good. @jvine and I just discussed the whole online content icon versus "online content" text label and we might want to revisit some aspects of how we do that in the future (beyond this workcycle, likely), but for now we think this is a good improvement we should move forward with.

**Pull request merged**